dashboard-ui/src/components/Results/resultsTable.jsx

```diff
@@ -0,0 +1,247 @@
+                                                performerADMArray.push({
+                                                    "value": performerADMOptions[i],
+                                                    "name": performerADMOptions[i]
+                                                });
+                                            }
+                                            performerADMArray.sort((a, b) => (a.value > b.value) ? 1 : -1);
+
+                                            return (
+                                                <List className="nav-list" component="nav" aria-label="secondary mailbox folder">
+                                                    {performerADMArray.map((item,key) =>
+                                                        <ListItem className="nav-list-item" id={"performeradm_" + key} key={"performeradm_" + key}
+                                                            button
+                                                            selected={this.state.adm === item.value}
+                                                            onClick={() => this.setPerformerADM(item.value)}>
+                                                            <ListItemText primary={this.formatADMString(item.value)} />
+                                                        </ListItem>
+                                                    )}
+                                                </List>
+                                            )
+                                        }
+                                    }
+                                    </Query>
+                                </div>
+                            </>
+                        }
+                    </div>
+                    <div className="test-overview-area">
+                        {(this.state.scenario !== "" && this.state.adm !== "") &&
+                            <Query query={test_by_adm_and_scenario} variables={{"scenarioID": this.state.scenario, "admName": this.state.adm}}>
+                            {
+                                ({ loading, error, data }) => {
+                                    if (loading) return <div>Loading ...</div> 
+                                    if (error) return <div>Error</div>
+                                    const testData = data[getTestByADMandScenarioName];
+
+                                    return (
+                                        <>
+                                            {testData !== null && testData !== undefined &&
+                                            <>
+                                                <AlignmentScoreBox performer={this.formatADMString(this.state.adm)} data={testData} scenario={this.state.scenario}/>
+                                                <div className='paper-container'>
+                                                    <TableContainer style={{ maxHeight: '70vh' }}>
```

Comment:
We should move to a css class.  (Know it's not yours but all the files are in the code review so checking them all)